### ISSUE TITLE
Cannot push image to container registry - 204 No Content

### ISSUE THREAD

**Issue by morsik** [goharbor/harbor#14201]
Not sure if bug or my misconfiguration, but I am unable to push any image to just-deployed Harbor:

Docker:
```console
morsik@xwing ~/Projects/etl-motd-rest $ sudo docker login https://harbor.mydomain.com
Authenticating with existing credentials...
Stored credentials invalid or expired
Username (admin): 
Password: 
WARNING! Your password will be stored unencrypted in /root/.docker/config.json.
Configure a credential helper to remove this warning. See
https://docs.docker.com/engine/reference/commandline/login/#credentials-store

Login Succeeded
morsik@xwing ~/Projects/etl-motd-rest $ sudo docker push harbor.mydomain.com/library/etl-motd-rest:fc30 
The push refers to repository [harbor.mydomain.com/library/etl-motd-rest]
e236e698f214: Pushing [==================================================>]  10.26MB/10.26MB
6a2fa25a1f50: Pushing [==================================================>]  53.44MB/53.44MB
b20db0943364: Pushing [==================================================>]  133.3MB/133.3MB
unknown blob
```

So, I tried Podman, and it gives a little more information:
```console
morsik@xwing ~/Projects/etl-motd-rest $ podman push harbor.mydomain.com/library/etl-motd-rest:latest
Getting image source signatures
Copying blob b9b4d8caaa77 [===>----------------------------------] 6.2MiB / 54.1MiB
Copying blob 1521f41ea3eb [==>-----------------------------------] 10.0MiB / 127.4MiB
Copying blob f8c91ab209a5 [==================================>---] 10.0MiB / 10.8MiB
Error: error copying image to the remote destination: Error writing blob: Error uploading layer to http://harbor.mydomain.com/v2/library/etl-motd-rest/blobs/uploads/0a44e5a3-b1a1-47ff-9964-d58556c0462c?_state=Cjy09EyrhQCnXEW2I0Z02Sex4LE7fZgP785wB_WVQX97Ik5hbWUiOiJsaWJyYXJ5L2V0bC1tb3RkLXJlc3QiLCJVVUlEIjoiMGE0NGU1YTMtYjFhMS00N2ZmLTk5NjQtZDU4NTU2YzA0NjJjIiwiT2Zmc2V0IjowLCJTdGFydGVkQXQiOiIyMDIxLTAyLTA3VDE1OjQ1OjU2WiJ9&digest=sha256%3Ae006437c19b33c5762e27a11eea172411d5ecbd34eac1fd33a7f123806dc0d05: received unexpected HTTP status: 204 No Content
```

But this is totally weird and I was unable to find any information about such issue.

I'm using CloudFlare as SSL reverse proxy in this simple case for now, so Harbor is configured only to serve 80 port, and I set `external_url` to be HTTPS as it's provided by CloudFlare, but apparently podman shows `http` in error which is weird.

Config I've used for installer (pretty original except domain is set, https disabled and password obviously changed):
```yaml
hostname: harbor.mydomain.com
http:
  port: 80
external_url: https://harbor.mydomain.com
harbor_admin_password: **REDACTED**
database:
  password: **REDACTED**
  max_idle_conns: 50
  max_open_conns: 1000
data_volume: /data
clair:
  updaters_interval: 12
trivy:
  ignore_unfixed: false
  skip_update: false
  insecure: false
jobservice:
  max_job_workers: 10
notification:
  webhook_job_max_retry: 10
chart:
  absolute_url: disabled
log:
  level: debug
  local:
    rotate_count: 50
    rotate_size: 200M
    location: /var/log/harbor
_version: 2.0.0
proxy:
  http_proxy:
  https_proxy:
  no_proxy:
  components:
    - core
    - jobservice
    - clair
    - trivy
```

**Comment by morsik** [comment]
Ok, I just found out that CloudFlare free tier has 100M POST limit.

Not sure how it's related because I have here 1 layer that's bigger than 100M, but 2 others are lower so… shouldn't they be uploaded at least?

Just in case: I verified Harbor working on port 80 and I added it to Insecure registries on both Docker and Podman and it works perfectly fine, so it must be something with CloudFlare in this case.
